src/core/structure.js

```diff
@@ -194,9 +192,12 @@ function filterHeader(h) {
 }
 
 export function run(conf) {
-  if ("tocIntroductory" in conf === false) {
-    conf.tocIntroductory = false;
+  if ("tocIntroductory" in conf) {
```

Comment:
I think we should only warn when `tocIntroductory` is false... people doing `tocIntroductory: true` are already doing the right thing.  

```suggestion
  if ("tocIntroductory" in conf && tocIntroductory === "false") {
```